.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,151 @@
+# Check for new versions of dask on Conda Forge
+name: WatchCondaForge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'pangeo-bot/pangeo-notebook-feedstock'
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          ref: 'master'
+        run: |
+          git config user.password ${{ secrets.PAT }}
+          git remote set-url origin https://github.com/pangeo-bot/pangeo-notebook-feedstock
+          git fetch --unshallow -p origin
```

Comment:
See https://github.com/peter-evans/create-pull-request/blob/master/docs/concepts-guidelines.md#push-pull-request-branches-to-a-fork

I think this is what we want but would be good to get a second set of eyes here.

Reply:
looks good. i *think* the workflow: 

1. checks out out https://github.com/conda-forge/pangeo-notebook-feedstock/master (so is synced up) 

2. then pushes changes to a fork branch https://github.com/pangeo-bot/pangeo-notebook-feedstock/upgrade-notebook-versions and finally 

3. creates a pull request upstream (conda-forge/pangeo-notebook-feedstock/master <-- pangeo-bot/pangeo-notebook-feedstock/upgrade-notebook-version)

@jhamman - can you add a comment describing why this offloading to pangeo-bot is necessary?

Reply:
see https://github.com/conda-forge/pangeo-dask-feedstock/issues/7